packages/kotti-ui/source/kotti-form/tests.ts

```diff
@@ -38,17 +38,11 @@ const TestField = defineComponent({
 	template: `<KtField :field="field" :getEmptyValue="() => null">FIELD</KtField>`,
 })
 
-const TestFieldObject = defineComponent({
+const TestFieldObject = defineComponent<KottiField.PropsInternal>({
```

Comment:
what even is a TestFieldObject??
is this a controller object?

if not, then why is the type `Record<string, unknown>` in the old code's DATA_TYPE